Loop/Views/SimpleBolusView.swift

```diff
@@ -0,0 +1,347 @@
+            }
+            .keyboardAware()
+            .edgesIgnoringSafeArea(self.isKeyboardVisible ? [] : .bottom)
+            .alert(item: self.$viewModel.activeAlert, content: self.alert(for:))
+            .alert(isPresented: self.$isClosedLoopOffInformationalModalVisible, content: self.closedLoopOffInformationalModal)
+        }
+    }
+    
+    private func shouldAutoScroll(basedOn geometry: GeometryProxy) -> Bool {
+        // Taking a guess of 640 to cover iPhone SE, iPod Touch, and other smaller devices.
+        // Devices such as the iPhone 11 Pro Max do not need to auto-scroll.
+        shouldBolusEntryBecomeFirstResponder && geometry.size.height < 640
+    }
+    
+    private var infoSection: some View {
+        HStack {
+            Image("Open Loop")
+            Text("When out of Closed Loop mode, the app uses a simplified bolus calculator like a typical pump.")
+                .font(.footnote)
+                .foregroundColor(.secondary)
+            infoButton
+        }
+    }
+    
+    private var infoButton: some View {
+        Button(
+            action: {
+                self.isClosedLoopOffInformationalModalVisible = true
+            },
+            label: {
+                Image(systemName: "info.circle")
+                    .font(.system(size: 25))
+                    .foregroundColor(.accentColor)
+            }
+        )
+    }
+    
+    private var summarySection: some View {
+        Section {
+            if displayMealEntry {
+                carbEntryRow
+            }
+            glucoseEntryRow
+            recommendedBolusRow
+            bolusEntryRow
+        }
+    }
+    
+    private var carbEntryRow: some View {
+        HStack(alignment: .center) {
+            Text("Carbohydrates", comment: "Label for carbohydrates entry row on simple bolus screen")
+            Spacer()
+            HStack {
+                DismissibleKeyboardTextField(
+                    text: $viewModel.enteredCarbAmount,
+                    placeholder: viewModel.carbPlaceholder,
+                    textAlignment: .right,
+                    keyboardType: .decimalPad
+                )
+                carbUnitsLabel
+            }
+            .padding([.top, .bottom], 5)
+            .fixedSize()
+            .modifier(LabelBackground())
+        }
+    }
+
+    private var glucoseEntryRow: some View {
+        HStack {
+            Text("Current Glucose", comment: "Label for glucose entry row on simple bolus screen")
+            Spacer()
+            HStack(alignment: .firstTextBaseline) {
+                DismissibleKeyboardTextField(
+                    text: $viewModel.enteredGlucoseAmount,
+                    placeholder: "---",
+                    // The heavy title is ending up clipped due to a bug that is fixed in iOS 14.
+                    font: .preferredFont(forTextStyle: .title1), // viewModel.enteredGlucoseAmount == "" ? .preferredFont(forTextStyle: .title1) : .heavy(.title1),
```

Comment:
[nit] remove unused code?

Reply:
The code is for future use with iOS 14.  I updated the comment to be clear about that.